bin/builder-core.js

```diff
@@ -38,6 +45,11 @@ module.exports = function (opts, callback) {
     runner: runner
   });
 
+  // Drain messages if any.
+  (opts.msgs || []).forEach(function (obj) {
```

Comment:
I could expose an `log.addQueue(opts.msgs)` and call before the `setLevel`, but I kind of like the manual draining here for the barrier between `builder.js` and `builder-core.js`